### PR TITLE
410 typedocs refactoring

### PR DESCRIPTION
**Pull request by Braunmann**

### CODE REVIEW

File: yajsapi/executor/executor.ts [hunk 1]
```diff
@@ -1,57 +1,50 @@
...
   package: string | Package;
-  /** Number of maximum parallel running task on one TaskExecutor instance */
-  maxParallelTasks?: number;
   /** Timeout for execute one task in ms */
-  timeout?: number;
-  /** TODO */
-  budget?: number;
-  /** Strategy used to choose best offer */
-  strategy?: MarketStrategy;
+  executorTimeout?: number;
```
Comment (mgordel) [inline yajsapi/executor/executor.ts:24]: `taskTimeout` will be more meaningful. We also need `executorTimeout`vfor all execution timeout

File: yajsapi/task/work.ts [hunk 2]
```diff
@@ -28,15 +28,15 @@ export interface WorkOptions {
 }
 
 export class WorkContext {
```
Comment (mgordel) [inline yajsapi/task/work.ts:30]: Maybe we should change the whole class name to `ProviderContext` ... ?

**Review by mgordel — APPROVED** [review]
I have some suggestions for improvements...

File: package.json [hunk 3]
```diff
@@ -69,6 +69,7 @@
     "eslint-config-prettier": "^8.5.0",
     "mocha": "^10.1.0",
     "node-stdlib-browser": "^1.2.0",
+    "npm-run-all": "^4.1.5",
```
[the remark this answers is not in the document]
Comment (Braunmann) [reply package.json:72]: true

**Pull request merged**